<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Punktwolke</string>
    </attribute>
    <widget class="QCheckBox" name="del_bottom">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>220</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Untergrund entfernen</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="del_upper">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>260</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Oberschicht entfernen</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="del_dorn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>300</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Dorn entfernen</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="rest">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>340</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Rest entfernen</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="rog">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>180</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Zugeschnitten auf Region of Interrest</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>10</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap>../iph_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="choose_pc">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Auswahl der Punktwolke</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>380</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Anwenden</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Datenverarbeitung</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="explain_data">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>260</y>
       <width>321</width>
       <height>271</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Zur Datenverarbeitung der Aufnahmen:&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-Alle zu verarbeiteten Daten müssen in dem Ordner Messungen liegen&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-Die Punktwolken müssen in den 3d-Ordner gepackt werden&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-Die Bilder dürfen in beliebig benannte Ordner innerhalb des Messungsordners liegen&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-vor dem Verarbeiteten der Daten muss mit den x und y Koordinaten das Schmiedeteil in die Mitte des Bildes gebracht werden&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>10</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap>../iph_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Daten/crop/crop1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="x_koor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>370</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
